model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -149,10 +159,14 @@ def __init__(
                     gm.GeometryName.EDGE_ORIENTATION_ON_VERTEX
                 ],
                 "vertex_owner_mask": gtx.as_field(
-                    (dims.VertexDim,), decomposition_info.owner_mask(dims.VertexDim)
+                    (dims.VertexDim,),
+                    decomposition_info.owner_mask(dims.VertexDim),
+                    allocator=self._backend,
                 ),
                 "cell_owner_mask": gtx.as_field(
-                    (dims.VertexDim,), decomposition_info.owner_mask(dims.CellDim)
+                    (dims.VertexDim,),
```

Comment:
```suggestion
                    (dims.CellDim,),
```

`dtype=bool` needed ?

Reply:
Thanks for this one. I don't know about the `dtype=bool` its a bool already. So I don't know whether gt4py needs it, but it can't harm.